internal/serverinstall/nomad.go

```diff
@@ -508,9 +508,10 @@ func waypointNomadJob(c nomadConfig) *api.Job {
 
 	task := api.NewTask("server", "docker")
 	task.Config = map[string]interface{}{
-		"image": c.serverImage,
-		"ports": []string{"server", "ui"},
-		"args":  []string{"server", "run", "-accept-tos", "-vvv", "-db=/alloc/data/data.db", fmt.Sprintf("-listen-grpc=0.0.0.0:%s", defaultGrpcPort), fmt.Sprintf("-listen-http=0.0.0.0:%s", defaultHttpPort)},
+		"image":          c.serverImage,
+		"ports":          []string{"server", "ui"},
+		"args":           []string{"server", "run", "-accept-tos", "-vvv", "-db=/alloc/data/data.db", fmt.Sprintf("-listen-grpc=0.0.0.0:%s", defaultGrpcPort), fmt.Sprintf("-listen-http=0.0.0.0:%s", defaultHttpPort)},
+		"auth_soft_fail": true,
```

Comment:
I think we should have users opt into this behavior, just so we aren't accidentally failing over to pulling a server image that wasn't intended by default. We can make this read from a flag, `-nomad-auth-soft-fail` where the default is false.

Reply:
Ok, i'll give that a try. 

Thank you so much for your help.

Reply:
Sure thing! There should be some good prior work in the serverinstall package for a simple bool flag just as a jumping off point.